Packs/DeepInstinct/Integrations/DeepInstinct/DeepInstinct.py

```diff
@@ -0,0 +1,222 @@
+import requests
+import json
+from datetime import datetime
+import demistomock as demisto
+
+from CommonServerPython import *
+
+requests.packages.urllib3.disable_warnings()
+
+
+def http_request(method, url_suffix, json=None):
+    """
+    Helper function to perform http request
+    """
+    base_url = demisto.params().get('base_url')
+    api_key = demisto.params().get('apikey')
+    api_suffix = "/api/v1"
+
+    headers = {
+        'Content-Type': 'application/json',
+        'Accept': 'application/json',
+        'Authorization': str(api_key)
+    }
+    r = requests.request(
+        method,
+        base_url + api_suffix + url_suffix,
+        json=json,
+        headers=headers,
+        verify=False
+    )
+
+    if r.status_code not in (200, 204):
+        s = 'Error in API call [%d] - %s ' % (r.status_code, r.reason)
+        return_error(message=s)
+    try:
+        return r.json()
+    except ValueError:
+        return None
+
+
+def get_specific_device():
+    """
+    Get specific device by id
+    """
+    device_id = demisto.args().get('device_id')
+    result = http_request('GET', "/devices/%s" % str(device_id))
+    ec = {'DeepInstinct.Devices(val.id && val.id == obj.id)': result}
+
+    return_outputs(
+        readable_output=tableToMarkdown('Device', result),
+        outputs=ec,
+        raw_response=result
+    )
+
+
+def add_hash_to_bl():
+    """
+    Add hash to blacklist
+    """
+    policy_id = demisto.args().get('policy_id')
+    file_hash = demisto.args().get('file_hash')
+    http_request('POST', '/policies/%s/blacklist/hashes/%s' % (str(policy_id), file_hash), json={"comment": ""})
+    demisto.results('ok')
```

Comment:
Please explain, didn't get such review in demo

Reply:
Eldad decided to dismiss this kind of comments in the meantime, but the point it that if you want to use the results from a previous command you have to see them in the context. otherwise there is no option to get them